gnupg/gnupg.py

```diff
@@ -369,13 +369,22 @@ def recv_keys(self, *keyids, **kwargs):
         """Import keys from a keyserver.
 
         >>> gpg = gnupg.GPG(homedir="doctests")
-        >>> key = gpg.recv_keys('hkp://pgp.mit.edu', '3FF0DB166A7476EA')
+        >>> key = gpg.recv_keys('3FF0DB166A7476EA',
+                                keyserver='hkp://pgp.mit.edu')
         >>> assert key
 
+        >>> ssl_keyserver = 'hkps://hkps.pool.sks-keyservers.net'
+        >>> ca_cert = '/home/user/hkps.pool.sks-keyservers.netCA.pem'
+        >>> gpg.recv_keys('6F682D87',
+                          keyserver=ssl_keyserver,
+                          keyserver_certs=ca_cert)
```

Comment:
I think we'll need `>>>` on these two lines to be able to run them as a doctest.

Reply:
Adding.